base/collection_xattr.go

```diff
@@ -402,6 +402,10 @@ func isKVError(err error, code memd.StatusCode) bool {
 	return false
 }
 
+func IsAlreadyExistsError(err error) bool {
+	return isKVError(err, memd.StatusKeyExists)
+}
```

Comment:
I think we've generally been removing these single-error functions haven't we? Or at least making them support all SDKs/buckets like `IsCasMismatch`